packages/shared/components/AccountTile.svelte

```diff
@@ -135,7 +150,6 @@
                 const timeNeeded = <number>getTimeUntilMinimumAirdropReward(account)
                 const remainingTime =
                     airdrop === StakingAirdrop.Assembly ? $assemblyStakingRemainingTime : $shimmerStakingRemainingTime
-
                 if (timeNeeded > remainingTime) {
```

Comment:
why not use the function canAccountReachMinimumAirdrop from staking.ts

Reply:
Because the data was needed later for the locale, however this could definitely be re-written.